README.md

```diff
@@ -17,14 +17,25 @@ GATE teamware provides a flexible web app platform for managing classification o
 * Deploy with [kubernetes](https://kubernetes.io/) or [docker compose](https://docs.docker.com/compose/).
 
 # Running the app
-## Requirements
-We recommend the following software as a minimum requirement for running GATE Teamware:
-* recommended OS: linux or macOS.
-* [git](http://git-scm.com/)
-* [docker](https://www.docker.com/) & [docker-compose](https://docs.docker.com/compose/)
-* [bash](https://www.gnu.org/software/bash/)
-
-## Instructions
+
+## Latest release
+
+The simplest way to deploy your own copy of GATE Teamware is to use Docker Compose on Linux or Mac.  Installation on Windows is possible but not officially supported - you need to be able to run `bash` shell scripts for the quick-start installer.
+
+1. Install Docker - [Docker Engine](https://docs.docker.com/engine/) for Linux servers or [Docker Desktop](https://docs.docker.com/desktop/) for Mac.
+2. Install [Docker Compose](https://github.com/docker/compose), if your Docker does not already include it (Compose is included by default with Docker Desktop)
+3. Download the [installation script](https://gate.ac.uk/get-teamware.sh) into an empty directory, run it and follow the instructions.
+
+```
+mkdir gate-teamware
+cd gate-teamware
+curl -LO https://gate.ac.uk/get-teamware.sh
```

Comment:
For now this is a file hosted directly on gate.ac.uk, but once we have a published release it will become a redirect to the latest GitHub release version.